src/mlpack/methods/ann/loss_functions/huber_loss_impl.hpp

```diff
@@ -30,13 +30,15 @@ HuberLoss<InputDataType, OutputDataType>::HuberLoss(
 
 template<typename InputDataType, typename OutputDataType>
 template<typename InputType, typename TargetType>
-double HuberLoss<InputDataType, OutputDataType>::Forward(
-    const InputType& input, const TargetType& target)
+typename InputType::value_type
+HuberLoss<InputDataType, OutputDataType>::Forward(const InputType& input,
+                                                  const TargetType& target)
 {
-  double loss = 0;
+  using RealType = typename InputType::value_type;
```

Comment:
In some other places we use: `typedef typename MatType::elem_type ElemType;` do you mind to change the style to be consistent with the rest of the codebase?

Reply:
Sure, @zoq. Also ```value_type``` -> ```elem_type```?

Reply:
Yes, looks like you already did that 👍 .